tests/snappi_tests/multidut/pfc/files/multidut_helper.py

```diff
@@ -42,21 +51,23 @@ def run_pfc_test(api,
                  test_traffic_pause,
                  snappi_extra_params=None):
     """
-    Run a multidut PFC test
+    Run a PFC test
```

Comment:
Revert to previous change i.e. multidut PFC test

Reply:
this has been fixed